### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [netbox-community/netbox#17515] [Help Wanted!] [answered]
Please help to solve the 500 error
I have 
Postgres 13
Python Version: 3.11.3
Netbox 4.1.1.
Nginx

When I click on some links, it does not open, in the browser console I see an error 500, but if I copy this link and immediately paste it into the browser, it opens without problems.

If I turn on debug, I see the following error.

![image](https://github.com/user-attachments/assets/2bc1c686-76f4-4295-bda3-62c6653b029c)

**Comment by SerhiiZahuba** [comment]
![image](https://github.com/user-attachments/assets/953d7d29-320d-44ea-99e9-f354bbf6c90e)

**Reply by candlerb** [reply, marked as answer]
Do you have "experimental HTMX navigation" turned on? If so, try turning it off. See #17313, #17397, #17506,

If that's not it, then try disabling plugins.

**Reply by SerhiiZahuba** [reply]
Thanks
